frontend/app/src/store/assets/asset-cache.ts

```diff
@@ -5,8 +5,8 @@ export const useAssetCacheStore = defineStore('assets/cache', () => {
 
   const { assetMapping } = useAssetInfoApi();
 
-  const { cache, isPending, retrieve, reset } = useItemCache<AssetInfo>(
-    async (keys: string[]) => {
+  const { cache, isPending, retrieve, reset, deleteCacheKey } =
```

Comment:
I don't think there is a need to expose the `deleteCacheKey` method, wouldn't the key be eventually pushed out because is unused anyway?

Reply:
Because I want to refresh the data, and invalidate the cache.
for example asset with id `a` is fetched and saved in the cache.
then I update the asset data for example the `name` or the `symbol`, I need to tell it to fetch the asset data from endpoint again.